packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -72,7 +72,7 @@ void main() {
     );
   }
 
-  final Function uiTestGroup = () {
+  final void Function() uiTestGroup = () {
```

Comment:
nit for here and elsewhere: maybe more terse to just call this `VoidCallback`.

Reply:
Done